test/integration/controlplane/transformation/kmsv2_transformation_test.go

```diff
@@ -163,6 +163,19 @@ func (r envelopekmsv2) plainTextPayload(secretETCDPath string) ([]byte, error) {
 	return plainSecret, nil
 }
 
+// TestDefaultValues tests default flag values without updating any of the feature flags
+func TestDefaultValues(t *testing.T) {
```

Comment:
Run the server with KMS v2 configured (without setting any feature gate or calling `SetKDFForTests`) and assert that the data stored in etcd is using KDF.